README.md

```diff
@@ -39,26 +39,22 @@ The Looker Embed SDK uses a fluent interface pattern. The construction of the em
 
 ### Building
 
-First initialize the SDK with address of your Looker server and, optionally, the endpoint on your server that will perform authentication.  (Note: Port must be included if it is required to reach the Looker server from browser clients, e.g. looker.example.com:443)
-These are used by all the embedded content.
+First initialize the SDK with address of your Looker server and the endpoint on your server that will perform authentication. (Note: Port must be included if it is required to reach the Looker server from browser clients, e.g. looker.example.com:443, but the protocol (http/https) should *not* be included.
```

Comment:
The example is confusing to me. It seems like 443 would be the one example where the port is NOT required to reach the Looker server from a browser client since it's the default port for https and is usually not provided

Reply:
agreed, I had copied this from the first readme but I made an update to show a case where you'd actually need a port included